packages/components/src/globals/scss/vendor/@carbon/elements/scss/themes/generated/_themes.scss

```diff
@@ -1460,12 +1508,28 @@ $carbon--theme--g100: (
     letter-spacing: 0,
   ),
   code-01: (
+    font-family: (
```

Comment:
Do these end up working with the type-style mixin? I didn't know it could process map values under the hood with `properties` mixin and that it would need to be a font family string

Reply:
yeah that's also part of the purpose of this PR (check slider type styles). this also affects components that were consuming `code-01` or `code-02` like date picker and time picker. they were calling `type-style` before but the font family was not included. now they should be included

Reply:
@emyarod I meant moreso that this line seemingly would emit the `map` value from `type-style`: https://github.com/carbon-design-system/carbon/blob/master/packages/type/scss/_styles.scss#L512 was surprised it worked and didn't see any updates to this mixin in the PR.